<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task1_Caesar</class>
 <widget class="QWidget" name="Task1_Caesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caesar Cipher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 18pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Caesar Cipher</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_shift">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Shift:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_shift">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0)</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Input Text:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter text here...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Output Text:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>500</width>
     <height>60</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Result will appear here...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_encrypt">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_decrypt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Back to Main</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
